internal/manifest/validation_test.go

```diff
@@ -0,0 +1,23 @@
+package manifest
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
```

Comment:
technically, this looks good to me.

I have a couple of questions, though:
* why are we using `testify`? do we use it in other packages?
* why is the name of this file `validation_test.go`? since it's being added as a test file for `validate_kubernetes.go`, wouldn't it be more appropriate to name it `validate_kubernetes_test.go`?

what confuses me is that there is also another file named `validate_test.go` (as a complement test file to `validate.go`) and `validation_test.go` oddly sticks out. 

[`internal/manifest` folder structure](https://github.com/Berops/claudie/tree/ea216b780c08af642e7a99540f97c2dada992f8b/internal/manifest)

Reply:
>why are we using testify? do we use it in other packages?

yes, in most of the test function.

>why is the name of this file validation_test.go? since it's being added as a test file for validate_kubernetes.go, wouldn't it be more appropriate to name it validate_kubernetes_test.go?

mb, have not noticed the `validate_test.go`

Reply:
> yes, in most of the test functions.

cool, right.
thanks.

> ..have not noticed..

no worries.